marlowe/changelog.d/20230515_083226_brian.bush_PLT_5596.md

```diff
@@ -0,0 +1,43 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Removed
+
+- A bullet item for the Removed category.
+
+-->
+<!--
+### Added
+
+- Implemented `marlowe-validators` executable that benchmark Plutus execution costs of the Marlowe validators.
+- Instrumeted `marlowe-test` to optionally generate benchmark cases.
+
+-->
```

Comment:
Don't forget to remove the block comment
```suggestion
### Added

- Implemented `marlowe-validators` executable that benchmark Plutus execution costs of the Marlowe validators.
- Instrumeted `marlowe-test` to optionally generate benchmark cases.

```

Reply:
I was wondering about that and didn't remove them because I thought it might confuse `scriv`. Now I know.